variables.tf

```diff
@@ -156,6 +156,7 @@ variable "enable_user_suffix" {
 variable "logs_destinations_ids" {
   type        = list(string)
   description = "List of destination resources Ids for logs diagnostics destination. Can be Storage Account, Log Analytics Workspace and Event Hub. No more than one of each can be set. Empty list to disable logging."
+  default     = []
```

Comment:
Having no default value here is a desired behavior.
We want the user to explicitly specify that he does not want any log instead of forget to set the value when needed.

Our modules are designed for production, and we should not have any case in production without logging activated.